src/server/routes/todoRouter.js

```diff
@@ -1,11 +1,12 @@
 const express = require("express");
-const { createItem, getItems, deleteItem } = require("./api");
+const { createItem, getItems, deleteItem, updateItem } = require("./api");
 const { validateSchema, todoSchema } = require("../middleware/validation");
 
 const todoRouter = express.Router();
 
 todoRouter.get("/items", getItems);
```

Comment:
server.js is not in the changelist, but don't forget to fix the static path that serves the dist folder